config/locales/en.yml

```diff
@@ -645,6 +645,7 @@ en:
       reported_with_application: Reported with application
       resolved: Resolved
       resolved_msg: Report successfully resolved!
+      sensitive_media_explanation: The reporting user has content warnings disabled or has chosen to always shows media, even if it contains sensitive content.
```

Comment:
I think the way you have this makes more sense, but just to say it out loud ... should this be *separate* notices for each condition (disable CW, show media) - or does this make sense to keep bundled since moderators are the audiences here and they understand why they are being shown this msg?

Reply:
I think combined is fine, it's really to say "the reporting user bypassed the content being normally hidden and therefore saw it"